e2e/testQA9-105.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { test, expect, Page } from "@playwright/test"
+import { PageManager } from "../page-objects/pageManager"
+import { product1 } from "../helper/data"
+
+let pm: PageManager
+
+test.describe('Items stay in cart after login or create account', () => {
+    let numOfProductsBefore: number;
+    let counterBefore: any;
+    test.beforeEach(async ({ page }) => {
+        pm = new PageManager(page)
+        await page.goto('/')
+        await expect(pm.onHomepage().storeLogo).toBeVisible() // Assert that the site is opened
+
+        // Call function to add product and capture returned values
+        const {
+            counterBefore: updatedCounterBefore,
+            numOfProductsBefore: updatedNumOfProductsBefore
+        } = await pm.onHomepage()
```

Comment:
Homepage please rename it to match with camelCase convention! (onHomePage)
Also rename the file homepage.ts to homePage.ts